packages/storage-plus/src/indexed_map.rs

```diff
@@ -295,8 +295,8 @@ mod test {
     }
 
     struct DataIndexes<'a> {
-        // Second arg is for storing pk
-        pub name: MultiIndex<'a, (String, String), Data, String>,
+        // Last args are for signaling pk deserialization
```

Comment:
Nitpicking, but for clarity - you mean type parameters? (in more places below as well)